tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -672,3 +671,66 @@ def test_dhcp_relay_on_dualtor_standby(ptfhost, dut_dhcp_relay_data, testing_con
     restart_dhcp_service(standby_duthost)
     pytest_assert(wait_until(120, 5, 0, check_interface_status, standby_duthost))
     pytest_assert(wait_until(120, 5, 0, check_interface_status, duthost))
+
+
+def test_dhcp_monitor_checksum_validation(ptfhost, dut_dhcp_relay_data, validate_dut_routes_exist, testing_config,
+                            setup_standby_ports_on_rand_unselected_tor,												# noqa F811
+                            rand_unselected_dut, toggle_all_simulator_ports_to_rand_selected_tor_m,    # noqa F811
+                            verify_acl_drop_on_standby_tor):     # noqa F811
```

Comment:
The `verify_acl_drop_on_standby_tor` is used to confirm how many packets are dropped from standby interfaces. I don't think it would directly pass in this case without any change, maybe we need add some support on this fixture